css-size-adjust-1/Overview.bs

```diff
@@ -44,6 +45,12 @@ Introduction {#intro}
   time, this needs to be done with minimal disruption to the overall
   design of the page.
 
+  Even on a display which is large enough to prevent horizontal scrolling,
```

Comment:
I think you're better off without this comma, i.e., "a display which is large enough to prevent horizontal scrolling when the user does not zoom into the page".

I also don't think "prevent" is quite the right word here.  Maybe "to prevent" -> "not to require", if that's what's intended?  If it's not what's intended, then maybe something else?